### DISCUSSION TITLE
Help with colors from grabber

### DISCUSSION THREAD

**Discussion by ashtonwalk** [awawa-dev/HyperHDR#751] [Q&A] [answered]
Hello, i am getting some weird colors when grabbing the video. Is there a way inside of HyperHDR I can correct this?

![shot164409](https://github.com/awawa-dev/HyperHDR/assets/157927068/26c94dbf-a015-4d9c-ae2e-b3faefc0f0af)

My issue seems to be resulting from trying to use EDID to enable Dolby Vision on my tv. The splitter i am using is capable of sending the EDID to my tv and not the raspberry pi zero i am using for HyperHDR. (My splitter has a toggle switch) This seems to be the only way i can keep Dolby Vision on my tv, but the hue when grabbing the video is off. If i disable EDID everything is fine the picture that the pi is receiving. I do understand that turning off EDID to my tv is probably the best solution but i then lose functionality of my new tv. I have used the grabber on a previous version of HyperHDR without issue. The only new equipment i have recently implemented is the splitter in order to get 4k picture quality on my tv. 

I am completely unfamiliar with LUT tables or calibration. I have attempted to try using these tools but with no luck. I am really unsure of what I am supposed to be doing. 

I would appreciate it if anyone could point me in the right direction or provide any tips!

**Comment by awawa-dev** [comment, marked as answer]
Hi
Take a look here: https://github.com/awawa-dev/HyperHDR/discussions/729#discussioncomment-8126042 (at the end)
Does it look familiar? Dolby Vision is a closed format. Currently, USB grabbers can only capture the its LLDV protocol - and only when forced to do so e.g. by an some HDMI splitters (via proper modified EDID). However, you need to select the equipment very carefully, especially the HDMI splitter, because not every device offers it. Since nothing is officially guaranteed, we only collect and share our experiences with this format using various equipment.

**Reply by ashtonwalk** [reply]
Thank you for the reply! 

Yes, that is almost exactly what I am dealing with! 

This is the splitter I am currently using: https://www.amazon.com/dp/B0CMZKGTGT?ref=ppx_yo2ov_dt_b_product_details&th=1

From reading your post it seems i need to go with the Ezcoo EZ-SP12H2.

Thank you for the help!

**Reply by ashtonwalk** [reply]
I just got the Ezcoo EZ-SP12H2 and everything is working perfectly! Thank you!!!
